starter-code/css/main.css

```diff
@@ -0,0 +1,76 @@
+body {
+  background-color: white;
+  font-family: "Lato", sans-serif;
+  padding-left: 35px;
+  border-left: 5px solid #bababa;
+
+}
+
+h1 a {
+  color: #f2c204;
+  font-size: 50px;
+  font-weight: 300;
+  text-decoration: none;
+}
+
+h2 {
+  font-weight: 200;
+  font-size: 25px;
+  padding-bottom: 0px;
+}
+
+h2 a {
+  color: #000000;
+  text-decoration: none;
```

Comment:
Since you've already written out this property, consider adding an anchor selector at the top of the file that clears all those text-decorations.
```css
a{ 
  text-decoration: none;
}
```